### DISCUSSION TITLE
Please delete this repo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#48615] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear GitHub staff

Hello, I want to solve a problem,[This repo](https://github.com/cirosantilli/china-dictatorship/) is violation of [Github terms of service](https://docs.github.com/zh/site-policy/github-terms/github-terms-of-service),open source licenses and [Criminal Law of the People's Republic of China](https://flk.npc.gov.cn/detail2.html?ZmY4MDgxODE3OTZhNjM2YTAxNzk4MjJhMTk2NDBjOTI=) Article 105,I'll have a few words for GitHub employees

First of all, I don't want GitHub to be completely banned by GFW, although there are many alternatives in China, but they can't be continued one by one, this is also the programmer's "home", we Chinese programmers don't want to be like Iranian programmers without "home" (referring to being sanctioned by the US government)

Secondly, GitHub is an open community, not a political tool, but in reality it is indeed a political tool, at present, GitHub has been banned by the US government in Iran, Russia and other countries, I don't want China in this list, but I believe GitHub does not want to have this list

Finally, I hope that those who engage in politics will not engage in politics in this open community and go elsewhere, otherwise the environment of our community will be polluted

Thank you!
Haoming,A Chinese programmer

**Comment by vedantmgoyal9** [comment]
cc @github-staff @github

**Comment by Akash1134** [comment]
Hi @hhm1   , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by ghost** [reply]
Hi, @Akash1134 ,It seems that these methods are useless, and I would like to get a satisfactory result by discussing the situation with GitHub staff
